python/full_bundle.py

```diff
@@ -173,8 +185,8 @@ def create(self) -> (int, "FullBundle"):
 
         # Find and dispatch to the appropriate BaseTechBundle.
```

Comment:
This comment is now stale. Please update.

Reply:
How come?

Reply:
Ping.

Reply:
Ping @porridge.

Reply:
It was not clear to me whether `BaseTechBundle` referred to the variable or class name, since they were the same here, right?

Reply:
It referred to the class name. The dispatching is still happening via [`BaseTechBundle.get_bundle_class(package_name)`](https://github.com/mesosphere/dcos-sdk-service-diagnostics/blob/c76280125b8f8a1b8703fac54c09e79e04a86930/python/base_tech_bundle/__init__.py#L150-L154).